classes/output/core_renderer.php

```diff
@@ -254,7 +254,7 @@ protected function render_badge_count() {
      * @return string
      */
     protected function render_messages() {
-        if ($this->page->theme->settings->messagestoggle == 0) {
```

Comment:
This causes an error when the Snap settings have not been saved - i.e. you have no settings stored.
You can probably replicate this by deleting the snap plugin (from the db, not the disk)